lib/stacks/next.js

```diff
@@ -54,10 +103,27 @@ async function next(projectName, projectPath, projectStyle) {
   output.success(`Folder and files created for ${output.cmd(projectName)}`);
 
   await fs.remove(sysPath.join(projectName, '.git'));
-  await install(projectName);
+  
+  const typescript = projectOption === 'typescript';
+  const root = sysPath.resolve(projectName);
+
+  await install(projectName, typescript);
   process.chdir(projectPath);
-  await initGit('Next');
 
+  if (typescript) {
+    await initForTypeScript(root);
+
+    const srcAllFiles = await recursiveReaddir(root, 'src');
+    const srcJsFiles = srcAllFiles.filter(file => new RegExp(/.js|jsx/).test(file));
+
+    for (const srcJsFile of srcJsFiles) {
+      const parsed = sysPath.parse(srcJsFile);
+      const newExt = parsed.ext === '.js' ? '.ts' : '.tsx';
+      await fs.rename(sysPath.join(root, srcJsFile), sysPath.join(root, `${parsed.dir}/${parsed.name}${newExt}`));
```

Comment:
I replaced all the JS files under the src directory with TS files, but I still get type errors at build time, as shown in the following image.

<img width="783" alt="スクリーンショット 2021-07-31 23 37 55" src="https://user-images.githubusercontent.com/38882716/127743384-38856da7-8f2e-41db-9f0e-4da50d75678e.png">

Reply:
This is an issue in the starter, we just need to set `const mesh = useRef(null)` I will do it.

Reply:
For the moment I fixed all the typescript issues if `strict:false`. In the future, if we want to handle `strict:true` we will need to add another brain on the starter. But this implies to maintain that branch which a big work. It's easy to change it to strict manually if the user really wants to I would prefer to let the user do it instead of maintaining another branch.